engine/apps/alerts/models/custom_button.py

```diff
@@ -48,15 +37,11 @@ class CustomButtonManager(models.Manager):
     def get_queryset(self):
         return CustomButtonQueryset(self.model, using=self._db).filter(deleted_at=None)
 
-    def hard_delete(self):
-        return self.get_queryset().hard_delete()
-
 
 class CustomButton(models.Model):
```

Comment:
we can't remove this model because of a foreign key reference [on the `AlertGroupLogRecord` model](https://github.com/grafana/oncall/blob/dev/engine/apps/alerts/models/alert_group_log_record.py#L209-L215) 🤔